crates/services/producer/src/block_producer/tests.rs

```diff
@@ -203,12 +210,44 @@ async fn production_fails_on_execution_error() {
     );
 }
 
+// TODO: Add test that checks the gas price on the mint tx after `Executor` refactor
```

Comment:
We can test for now that we are passing the gas price from the GP to the executor.